src/st/wrp/data.c

```diff
@@ -0,0 +1,10 @@
+#include "types.h"
+
+/* 0x80193AB0 */ s32 g_LayoutObjHorizontal;
```

Comment:
There is no need to have the offset as a comment. We can retrieve it any time by looking at `build/us/stwrp.map`